.gitignore

```diff
@@ -6,6 +6,7 @@ local_settings.py
 archweb.db
 archweb.db-*
 database.db
+/*.tar.gz
```

Comment:
Please move this to a separate commit (Can even be a separate PR which we can already merge)

Reply:
Sure :)

Reply:
I just noticed there was already a `*.tar.gz` that doesn't trigger on tar files in the root directory. I'll tweak that line instead of adding another one.

Reply:
https://github.com/archlinux/archweb/pull/449